### DISCUSSION TITLE
Suggestion: Implementation of Country Codes, Country Sub Division codes with Country Sub Division input like Radio-Browser

### DISCUSSION THREAD

**Discussion by goliv04053** [jcorporation/webradiodb#459] [General]
I think it would be interesting to implement these features similar to the Radio-Browser. It could be used for example to categorize radios from certain countries by their regions.

For example, Radio Grenal is a radio from Rio Grande do Sul and FM O DIA is a music radio from Rio de Janeiro.

**Comment by jcorporation** [comment]
I must see what can be implemented. The GitHub formular is limited in the feature set, but some additional fields are possible.

Other idea could be the implementation of a custom formular that opens the github issue in a second step.

However, I am unsure if the additional information is really useful.

**Reply by goliv04053** [reply]
Example: I want to see only the radios from my state or region. I do not want to scroll to various radios from various regions from my country. Implementing country sub divisions will improve the experience.

**Reply by jcorporation** [reply]
I think an additional state field that is linked with the country field is reasonable.

**Reply by goliv04053** [reply]
Yes

**Reply by jcorporation** [reply]
The additional state field is now in the backend and in the issue forms.  I hope I have time to extend the station search tomorrow.

**Reply by jcorporation** [reply]
The station search is now extended.
